workflow/environments/org/nesi/create_python_virtenv_mahuika.sh

```diff
@@ -34,3 +34,9 @@ pip install -e ./qcore
 pip install -e ./Empirical_Engine
 pip install -I --no-deps -e ./IM_calculation
 pip install -e ./visualization
+
+cd ${env_path}/qcore
+python setup.py install
```

Comment:
For the qcore install? should we update the previous install line above rather than installing it twice.